chalicelib_fourfront/accounts.json

```diff
@@ -0,0 +1 @@
+9NovjmdNf+ITE/xELDWvv8CYlsXKvWDEy3MXV4fk5VvRX0fBWTRuvrRX5W8KzwvZGCzdqfzIPjrrEPWWMaDjEbuFl9aZ1uZJSO7NigVjl6yczgww2Vj0/HlnG75vsoeES5lLMv0BoQYi8lpWxHJJWOQHbYUq1EAkAknzJKRUemFbYPVwu2NADOXLOReCVvlzgJpvfERwpFU4wFghf/q3LYHD03PivsrcH+A9yFvc+b2MGzhZGAwfjrk7X/l8f+kf4xVFSaXvaU2s5yEHgfsWwKQt/9mPPuCMu4WX1pnW5klI7s2KBWOXrJzODDDZWPT8Nbkki4cO7mfZDqF6CB8p4CLyWlbEcklY5AdthSrUQCQCSfMkpFR6YVtg9XC7Y0AM5cs5F4JW+XNJCDhSsCEzUvrkWeFmBdKR
```

Comment:
I would favor this file being called `accounts.json.encrypted`.  Or something like that. I don't see how the contents of this file are JSON.